packages/logs/src/logs.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  areCookiesAuthorized,
+  Batch,
+  combine,
+  commonInit,
+  Configuration,
+  Context,
+  ErrorMessage,
+  getTimestamp,
+  HttpRequest,
+  mustUseSecureCookie,
+} from '@datadog/browser-core'
+import { buildEnv } from './buildEnv'
+import { Logger, LogsMessage } from './logger'
+import { LoggerSession, startLoggerSession } from './loggerSession'
+import { LogsUserConfiguration } from './logs.entry'
+
+export function startLogs(
+  userConfiguration: LogsUserConfiguration,
+  errorLogger: Logger,
+  getGlobalContext: () => Context
+) {
+  const isCollectingError = userConfiguration.forwardErrorsToLogs !== false
+  const logsUserConfiguration = {
+    ...userConfiguration,
+    isCollectingError,
+  }
```

Comment:
I think it would be better to completly `isCollectingError` from the configuration. Each SDK are overriding their configuration to set this value before passing it to `commonInit`. It would be much simpler to pass it as a second argument to `commonInit`.
WDYT?

Reply:
I am assuming it is

> to completely **remove** `isCollectingError`

Yeah, it could be nice